src/server/services/wfs/qgswfsgetfeature.cpp

```diff
@@ -464,7 +464,8 @@ namespace QgsWfs
         const QString srsName {request.serverParameters().value( QStringLiteral( "SRSNAME" ) )};
         const bool invertAxis { mWfsParameters.versionAsNumber() >= QgsProjectVersion( 1, 1, 0 ) &&
                                 outputCrs.hasAxisInverted() &&
-                                ! srsName.startsWith( QLatin1String( "EPSG:" ) ) };
+                                ! srsName.startsWith( QLatin1String( "EPSG:" ) ) &&
```

Comment:
The new behavior makes sense to me, but I would ask you to explore a different implementation which consists in setting `srsName` to the advertized default when it was not explicitly set by `SRSNAME` query string parameter. This way you don't have to add another check for the empty string here.

Reply:
I have updated the PR. Is this what you had in mind?

Reply:
@ptitjano yes that's more or less what I had in mind: I was thinking to get the default crs from the list of advertized CRSs (i.e. the first CRS listed in getcapabilities as the available CRSs), you chose to get it from `outputCrs` which if I read the code correctly defaults to the layer's CRS and it might be wrong.

It seems wrong to me because what happens if the layer's CRS is not one of the advertized ones?

Can you please check if in case of:
- layer's CRS is not one of the advertized CRSs in getcapabilities
- SRSNAME is not passed in the query string

the output is the expected one (i.e. the first default CRS from the list of advertized ones) ? 

Unless I am forgetting something about the expected CRS in case of no SRSNAME: I didn't check the specifications recently.